xbmc/addons/kodi-addon-dev-kit/include/kodi/xbmc_pvr_dll.h

```diff
@@ -519,6 +519,15 @@ extern "C"
    *  @remarks This will only be used if the backend doesn't provide a direct URL in the recording tag.
    */
   //@{
+  /*!
+   * Obtain the chunk size to use when reading streams.
```

Comment:
we should note that this has no effect when reading from demuxer

Reply:
Okay, will add this to the `@remarks` section for this function.

Reply:
done